lsp/intellij-plugin/src/main/kotlin/Icons.kt

```diff
@@ -0,0 +1,9 @@
+// Copyright 2000-2022 JetBrains s.r.o. and other contributors. Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE file.
+package community.flock.wire_spec.lsp.intellij_plugin
+
+import com.intellij.openapi.util.IconLoader
+import javax.swing.Icon
+
+object Icons {
```

Comment:
if its only used in `FileType` move it there. Maybe even in companion object

Reply:
I follow the intellij example here. Like to stay close to it